eclass/kernel-build.eclass

```diff
@@ -113,6 +113,12 @@ IUSE="+strip"
 # root=/dev/gpt-auto-root ro
 : "${KERNEL_GENERIC_UKI_CMDLINE:="root=/dev/gpt-auto-root ro"}"
 
+# @ECLASS_VARIABLE: KERNEL_VERBOSE
+# @USER_VARIABLE
+# @DESCRIPTION:
+# Set to OFF to disable verbose messages during compilation
+: "${KERNEL_VERBOSE:=ON}"
```

Comment:
Mixed feelings. On one hand, it seems consistent with (most) other eclasses. On the other, the kernel itself uses 0/1 here.